src/main/java/org/llorllale/cactoos/matchers/TextIsEmpty.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) for portions of project cactoos-matchers are held by
+ * Yegor Bugayenko, 2017-2018, as part of project cactoos.
+ * All other copyright for project cactoos-matchers are held by
+ * George Aristy, 2018-2020.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.llorllale.cactoos.matchers;
+
+import org.cactoos.Text;
+
+/**
+ * Matches if a text <em>isEmpty</em>.
+ *
+ * @since 1.0.0
+ */
+public final class TextIsEmpty extends MatcherEnvelope<Text> {
+
+    /**
+     * Ctor.
+     */
+    public TextIsEmpty() {
+        super(
+            new MatcherOf<>(
```

Comment:
@kokodyn Can you use `TextMatcherEnvelope` instead?

Reply:
@andreoss do you wish me to create such a "TextMatcherEnvelope " class?

Reply:
@kokodyn It was renamed to `TextMatcher` 88eb534435e4a4a2d84be1684045014074ec37d5

Reply:
@andreoss @kokodyn what about this?